crates/sui-graphql-rpc/src/context_data/pg_backend.rs

```diff
@@ -640,10 +640,7 @@ impl PgQueryExecutor for PgManager {
                                 exceeds = true
                             );
                         } else {
-                            info!(
```

Comment:
I think your editor redid some stuff here, you probably did not want to modify this file.